22:56:25 [INF][   WBContext UI]: Initializing workbench context UI with these values:
	base dir: /usr/share/mysql-workbench
	plugin path: /usr/lib/mysql-workbench/plugins
	struct path: /usr/share/mysql-workbench/grt
	module path: /usr/lib/mysql-workbench/modules
	library path: /usr/share/mysql-workbench/libraries
	user data dir: /config/.mysql/workbench
	open at start: 
	open type: 
	run at startup: 
	run type: 
	Force SW rendering: No
	Force OpenGL: No
	quit when done: No
22:56:25 [INF][      WBContext]: WbContext::init
22:56:25 [INF][            WBA]: Looking for extension modules for WBA...
22:56:25 [INF][            WBA]: 0 extension modules found
22:56:25 [WRN][            grt]: Duplicate plugin name wb.tools.cmdlineClient	There is more than one plugin with the name wb.tools.cmdlineClient (in PyWbUtils and PyWbUtils).
22:56:25 [WRN][            grt]: /config/.mysql/workbench/server_instances.xml:5: link 'e3f512c2-3af4-11ee-a6f9-0242ac140003' <object db.mgmt.Connection> key=connection could not be resolved
22:56:25 [WRN][            grt]: /config/.mysql/workbench/server_instances.xml:17: link '9f537c44-3af4-11ee-a6f9-0242ac140003' <object GrtObject> key=owner could not be resolved
22:56:25 [WRN][            grt]: Duplicate plugin name wb.tools.cmdlineClient	There is more than one plugin with the name wb.tools.cmdlineClient (in PyWbUtils and PyWbUtils).
22:56:25 [INF][      WBContext]: System info:
 	MySQL Workbench Community (GPL) for Linux/Unix version 8.0.34 CE build 3263449 (64 bit)
	Configuration Directory: /config/.mysql/workbench
	Data Directory: /usr/share/mysql-workbench
	Cairo Version: 1.16.0
	OS:  x86_64
	CPU: 8x 11th Gen Intel(R) Core(TM) i5-11300H @ 3.10GHz (3100.000MHz) - 15.33GiB RAM
No video adapter info available
	Distribution: Ubuntu 22.04.2 LTS

	Fips mode enabled: no

22:56:25 [INF][    WQE backend]: Found auto-save workspace /config/.mysql/workbench/sql_workspaces/-1.autosave
22:56:25 [WRN][       WBModule]: OS not found on supported OS list. OS string: ' x86_64'
22:58:02 [ERR][SQL Editor Form]: SqlEditorForm: exception in do_connect method: Exception: Unable to connect to localhost
22:58:02 [WRN][SQL Editor Form]: Unable to connect to localhost
22:58:02 [INF][SQL Editor Form]: Error 2003 connecting to server, assuming server is down and opening editor with no connection
22:58:02 [INF][SQL Editor Form]: Error 2003 connecting to server, assuming server is down and opening editor with no connection
22:58:02 [INF][          Admin]: Auto-created instance profile for connection 
22:58:02 [ERR][wb_admin_control.py:init:307]: Error connecting to MySQL: Unable to connect to localhost (code 2003)
